vm-builder/main.go

```diff
@@ -541,3 +564,15 @@ func (f file) validate() []error {
 
 	return errs
 }
+
+func getAgettyInitLine(targetArch string) string {
+	switch targetArch {
+	case targetArchLinuxAmd64:
+		return scriptAgettyInitAmd64
```

Comment:
Why did you put it into a separate file, instead of constructing the line here? The bulk of it can be put as a string literal here, and the final string gets formatetted based on the architecture. 

A description or a comment explaining why agetty line needs to be different would be helpful also.

Reply:
I didn't want to use string literal, because the line was quite long. Though this change actually comes from the time I wasn't sure how much difference we might want between agetty for arm and amd, currently it can be replaced by passing only tty variable, since it is the only difference.